landscaper/common/generate/openapi/generate.go

```diff
@@ -56,10 +56,10 @@ func init() {
 		"the output path for the generated OpenAPI code")
 
 	rootCommand.Flags().StringVar(
-		&inputDirectory,
-		"input-directory",
+		&inputDirectories,
+		"input-directories",
 		"",
-		"the absolute input directory which should contain the go types to generate OpenAPI code for. Example: /Users/<superuser>/go/src/github.com/gardener/gardener/landscaper/pkg/controlplane/apis/imports/v1alpha1")
+		"the absolute input directories (colon-seperated) which should contain the go types to generate OpenAPI code for. Example: `/Users/<superuser>/go/src/github.com/gardener/gardener/landscaper/pkg/controlplane/apis/imports/v1alpha1,other-directory`")
```

Comment:
Do the backticks contribute to something in this help text? If not, I would suggest to remove them from the help text.